website/pages/docs/[[...page]].jsx

```diff
@@ -19,29 +16,28 @@ export default function DocsLayout(props) {
       baseRoute={basePath}
       staticProps={props}
       additionalComponents={additionalComponents}
+      showVersionSelect={process.env.ENABLE_VERSIONED_DOCS === 'true'}
```

Comment:
This shouldn't be necessary 💭 

per: https://github.com/hashicorp/react-components/blob/main/packages/docs-page/index.tsx#L52